helidon-ls-ext/helidon.jdt.ext/helidon.jdt.ext.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
```

Comment:
Why is the manifest hard-coded and not generated ?

Reply:
it is used by Tycho